store/demo/dev/10220__sheet.sql

```diff
@@ -18,10 +16,69 @@ VALUES
         101,
         3001,
         7019,
-        'my-sheet',
-        'SELECT * FROM demo',
-        'BYTEBASE',
-        'SQL'
+        'demo data',
+        'SELECT * FROM demo'
+    );
+
+INSERT INTO
+    sheet (
+        id,
+        creator_id,
+        updater_id,
+        project_id,
+        name,
+        statement
+    )
+VALUES
+    (
+        22002,
+        101,
+        101,
+        3002,
+        'all employee',
+        'SELECT * FROM employee'
```

Comment:
nit: if possible, use a slight variation in the statement for public/project/private sheet to be more readable